ctrd/utils.go

```diff
@@ -125,3 +128,27 @@ func convertCtrdErr(err error) error {
 
 	return err
 }
+
```

Comment:
Can this part move to ctrd/snapshotter_key.go

Reply:
Move it !